templates/pages/reset-password.jsx

```diff
@@ -78,16 +81,19 @@ var ResetPassword = React.createClass({
         redirectObj.query.state = queryObj.state;
         redirectObj.query.response_type = queryObj.response_type;
         redirectObj.query.scopes = queryObj.scopes;
+        ga.event({category: 'Reset Password', action: 'Successfully request new password'});
         window.location = Url.format(redirectObj);
         return;
       }
       // handle errors!
     }.bind(this)).catch(function(ex) {
+      ga.event({category: 'Reset Password', action: 'Error parsing response from the server'});
       console.error('Error parsing response', ex);
     });
   },
   handleResetPassword: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Reset Password', action: 'Error form validation on reset password page'});
```

Comment:
Also ignore this comment - as the specific field validation is captured by `form.jsx`